### ISSUE TITLE
URL argument buggy on windows

### ISSUE THREAD

**Issue by nothub** [nothub/mrpack-install#30]
Reported on Discord:
```
PS C:\Users\User\AppData\Roaming\Minecrat_Servers\CFC> mrpack-install "https://cdn.modrinth.com/data/zj6Ob29g/versions/tt6mZDVe/Chronicles%20From%20Creation%201.0.1.mrpack"
2023/10/30 22:56:58 CreateFile https://cdn.modrinth.com/data/zj6Ob29g/versions/tt6mZDVe/Chronicles%20From%20Creation%201.0.1.mrpack: The filename, directory name, or volume label syntax is incorrect.
```

**Comment by nothub** [comment]
same issue as #27

**Comment by nothub** [comment]
fixed by d9f2f244ff38c78890936732b742680c84d9ab14
fix released in [0.16.9](https://github.com/nothub/mrpack-install/releases/tag/v0.16.9)
